tests/snapshots/install_empty_config_file.snap

```diff
@@ -0,0 +1,9 @@
+---
+source: tests/cli.rs
+assertion_line: 94
+expression: content
+
+---
+thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error { inner: ErrorInner { kind: Custom, line: None, col: 0, at: None, message: "missing field `tools`", key: [] } }', src/config.rs:97:69
+note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Comment:
A note on this test (and tests/snapshots/install_invalid_tool_configuration.snap): I've let these in even if they're not great just to document the current behavior. I have some changes locally that I'll put up once this merges that refactor the error handling to get better output.